### ISSUE TITLE
there is a problem when run 

### ISSUE THREAD

**Issue by fakefish** [arnif/redux-react-native-counter-example#1]
```
TypeError: Cannot read property 'root' of null
    at /Users/fakefish/Documents/coding/R-React/redux/redux-react-native-counter-example/node_modules/react-native/packager/react-packager/src/DependencyResolver/crawlers/index.js:16:84
    at tryCallOne (/Users/fakefish/Documents/coding/R-React/redux/redux-react-native-counter-example/node_modules/react-native/node_modules/promise/lib/core.js:37:12)
    at /Users/fakefish/Documents/coding/R-React/redux/redux-react-native-counter-example/node_modules/react-native/node_modules/promise/lib/core.js:103:15
    at flush (/Users/fakefish/Documents/coding/R-React/redux/redux-react-native-counter-example/node_modules/react-native/node_modules/promise/node_modules/asap/raw.js:50:29)
    at process._tickCallback (node.js:355:11)
```

what happened?

**Comment by arnif** [comment]
Hmm good question. What version of watchman do tou have ?

**Comment by fakefish** [comment]
@arnif 3.0.0

**Comment by arnif** [comment]
ok, upgrade to [3.3.0](https://facebook.github.io/watchman/docs/release-notes.html#watchman-3-3-0-2015-06-22) and try again.

**Comment by arnif** [comment]
@fakefish did that solve the issue ?

**Comment by fakefish** [comment]
@arnif I solved this issue by update watchman to 3.7.0, it's awsome!

**Comment by arnif** [comment]
Good to hear :smile:
